bigbluebutton-html5/imports/ui/components/end-meeting-confirmation/component.jsx

```diff
@@ -40,6 +40,8 @@ class EndMeetingComponent extends React.PureComponent {
       users, intl, closeModal, endMeeting, meetingTitle,
     } = this.props;
 
+    const message = intl.formatMessage(intlMessages.endMeetingDescription, { 0: users }).split('.');
```

Comment:
I like the shortcut you took via `split('.')` but things can go bad if a translation string contains multiple '.'.

More resilient approach would be to split the end meeting description into two parts and go from there

Reply:
applied suggested changes in 0a09112